src/ucp/api/ucp.h

```diff
@@ -1878,6 +1981,85 @@ ucs_status_ptr_t ucp_tag_send_sync_nb(ucp_ep_h ep, const void *buffer, size_t co
 
 /**
  * @ingroup UCP_COMM
+ * @brief Non-blocking stream receive operation.
```

Comment:
Suggest the following text for clarity:
 Non-blocking stream receive operation of structured data into a user-supplied buffer.